### ISSUE TITLE
Genetic code start codon variation

### ISSUE THREAD

**Issue by cmayer** [bbuchfink/diamond#773]
Some genetic codes such as the Vertebrate Mitochondrial Code allow potentially different start codons / start amino acids. Is it possible to take this into account in diamond, e.g. by specifying that the reference sequence starts with such a start codon?

**Comment by bbuchfink** [comment]
No, you could only filter you database yourself for such sequences.
